user_tools/src/spark_rapids_pytools/cloud_api/dataproc_gke.py

```diff
@@ -42,6 +42,7 @@ class DataprocGkePlatform(DataprocPlatform):
     def __post_init__(self):
         super().__post_init__()
         self.type_id = CspEnv.DATAPROC_GKE
+        self.cluster_inference_supported = False
```

Comment:
why don't we support inference here?

Reply:
I did not have clarity on the template cluster config for dataproc-gke. I will try it again when shifting these templates to moustache

Reply:
I think we should have a separate PR for dataproc gke. Dataproc Gke uses 2 layer. We need to create an object of dataproc gke followed by an object of dataproc. This could increase the complexity of this PR

Reply:
Works for me.